src/EnergyPlus/MixedAir.cc

```diff
@@ -3061,37 +3053,39 @@ namespace MixedAir {
 
 				// Make sure all zones with mechanical ventilation are on the correct air loop
 				TempMechVentArrayCounter = 0;
+				int AirLoopZoneInfoSize = static_cast <int> ( AirLoopZoneInfo.size() );
```

Comment:
@JasonGlazer If I remember correctly, I had suggested earlier that you only need to check for Airloopnum > 0.  Was that not valid?  I don't think checking size is the right fix here.